src/fixup.js

```diff
@@ -139,4 +140,57 @@
     }
   }
 
+  /* Deprecation warning */
+  var request = new XMLHttpRequest();
+
+  request.open('GET', '//www.w3.org/TR/tr-outdated-spec');
+  request.onload = function() {
+    if (request.status < 200 && request.status > 400) {
+      return;
+    }
+    try {
+      var currentSpec = JSON.parse(request.responseText);
+    } catch (err) {
+      console.error(err);
+      return;
+    }
+    document.body.classList.add('outdated-spec');
+    var node = document.createElement("p");
+    node.classList.add("outdated-warning");
+
+    var warning =
```

Comment:
I'm still worried that this is quite unmaintainable an fragile - maybe something like:

```JS
var heading = "<strong>" + currentSpec.header + "</strong>";
var anchor = "<a id=outdated-note href=" + currentSpec.latestUrl + ">" + currentSpec.latestUrl + "</a>.";
var warning = "<span>" + currentSpec.warning + "</span> " + anchor;
var button = "<input onclick='collapseWarning(false)' type=button value='&#9662; collapse'>";
var html = heading + warning + button;  
node.innerHTML = html;
```

Or, even:
```JS
var frag = document.createDocumentFragment();
var heading = document.createElement("strong");
heading.innerHTML = currentSpec.header;
frag.appendChild(heading);

var anchor = document.createElement("a");
anchor.id = "outdated-note";
anchor.href = currentSpec.latestUrl;
anchor.innerText = currentSpec.latestUrl + ".";
frag.appendChild(anchor);

var warning = document.createElement("<span>");
warning.innerText = currentSpec.warning;
warning.appendChild(anchor);
frag.appendChild(warning);

var button = document.createElement("button");
button.addEventListener("click", function() {
  collapseWarning(false);
});
button.innerHTML = "&#9662; collapse";
frag.appendChild(anchor);
node.appendChild(frag);
```

Might make this a bit more maintainable. 

(note that I just wrote the above out without actually checking it... so if you use either, please confirm that I didn't screw up the HTML structure)